otseobande/maze-problem
# Implement move sounds

**What does this PR do?**

This PR adds sounds on play move

**Description of Task to be completed?**

- add sound when mario moves
- add sound when mario collects a mushroom
- add sound when game is completed
- write new tests and refactor old tests

**How should this be manually tested?**

- Fetch and pull this branch
- Start the server by running `$yarn start`
- Move the player with your arrow keys
- The browser should make sounds on move

**Any background context you want to provide?**

- N/A

**What are the relevant pivotal tracker stories?**

- #162331297

**Screenshots (if appropriate)**

N/A

**Questions:**

N/A
